Patches/The GiantRace/ThingDefs_Misc/GiantApparel.xml

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+  <Operation Class="PatchOperationSequence">
+	<operations>
+	  <li Class="PatchOperationFindMod">
+			
+		<mods><li>The GiantRace</li></mods>
+			
+		<match Class="PatchOperationSequence">
+		<operations>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="GI_GiantRags"]/statBases/StuffEffectMultiplierArmor</xpath>
+				<value>
+					<StuffEffectMultiplierArmor>5</StuffEffectMultiplierArmor>
+					<Bulk>6</Bulk>
+					<WornBulk>0</WornBulk>
```

Comment:
Why zero, rather than 1, as usual?

Reply:
As it turns out regular clothes actually have 0 worn bulk, so my sleep deprived brain gave it 0 rather than just removing the field. It doesn't really matter since the net effect is the same either way and you can physically see it here, but I can remove it if you'd like.